pkgs/applications/networking/browsers/firefox/packages.nix

```diff
@@ -61,4 +61,37 @@ rec {
       versionKey = "ffversion";
     };
   };
+
+  firefox-esr-68 = (common rec {
+    pname = "firefox-esr";
+    ffversion = "68.11.0esr";
+    src = fetchurl {
+      url = "mirror://mozilla/firefox/releases/${ffversion}/source/firefox-${ffversion}.source.tar.xz";
+      sha512 = "0zg41jnbnpsa07xaizwfsmfav0cgxdqnh8i4yanxy49a45gigk895zqrx2if7pfsmdnj9zpwj9prj8cpnpsfhv6p62f3g2596aa9kvx";
+    };
+
+    patches = [
+      ./no-buildconfig-ffx65.patch
+    ];
+
+    meta = firefox.meta // {
+      description = "A web browser built from Firefox Extended Support Release source tree";
+    };
+    updateScript = callPackage ./update.nix {
+      attrPath = "firefox-esr-68-unwrapped";
+      versionSuffix = "esr";
+      versionKey = "ffversion";
+    };
+  }).override {
+    # Mozilla unfortunately doesn't support building with latest NSS anymore;
+    # instead they provide ESR releases for NSS:
+    # https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS/NSS_Releases
+    nss = nss_3_44;
+  };
+
+  # apply enterprise-policies.patch so that we can have enterprise policies
+  # on a pr. profile basis, instead of pr. user or global for the installation.
+  firefox-policies = firefox.overrideAttrs(old: {
```

Comment:
Is there any downside from having this in our default Firefox build? From the patch above I can see that we can select the mode during runtime anyway.